docs/configuration.md

```diff
@@ -137,6 +137,13 @@ statistics:
 # Export the metric with the original CloudWatch timestamp (General Setting for all metrics in this job)
 [ addCloudwatchTimestamp: <boolean> ]
 
+# Enables the inclusion of past metric data points from the CloudWatch response if available.
+# This is useful when a metric is configured with a 60-second period and a 300-second duration, ensuring that all
+# five data points are exposed at the metrics endpoint instead of only the latest one.
+# Note: This option requires `addCloudwatchTimestamp` to be enabled
```

Comment:
I think it will also require that the metrics destination you are using has support for out of order timestamps. Probably worth a call out to at least the prometheus docs about it https://prometheus.io/docs/prometheus/latest/configuration/configuration/#tsdb.